source/configuring_domain/05_configuring_domain/regional_settings.rst

```diff
@@ -18,3 +18,20 @@ To change the timezone of a domain, go to Configuration > Regional settings. In
    :class: note
 
    It is also possible to directly write a city or zone in the textbox. A selection of matching existing timezones will appear.
+
+
+CSV delimiter
+-------------
+
+The preferred delimiter for CSV files, usually depending on geographical location, can either be a comma or a semicolon. The CSV delimiter configuration allows to define what delimiter will be used by default for datasets exports in CSV format.
+
+.. admonition:: Note
+   :class: note
+
+   This configuration does not apply to the API, which has its own way of choosing the delimiter for CSV exports (see API Search v1 and `API Search v2 <https://help.opendatasoft.com/apis/ods-search-v2/#exporting-datasets>`_).
+
+To define the default delimiter for CSV exports, go to Configuration > Regional settings. In the subsection of the back office is displayed the CSV delimiter configuration.
```

Comment:
This sentence sounds weird IMO. "The CSV delimiter configuration is displayed in a subsection of the page"?